packages/apps/human-app/frontend/src/modules/worker/hooks/use-register-address/use-register-address-validation.ts

```diff
@@ -0,0 +1,14 @@
+import { useWalletConnect } from '@/shared/hooks/use-wallet-connect';
+
+export function useRegisterAddressValidation() {
+  const { address } = useWalletConnect();
+
+  const errors = {
+    noAddress: !address,
```

Comment:
Why making `errors` object with specific properties instead of e.g. array with error messages?